### ISSUE TITLE
Adding additional metrics to fine-tuning output

### ISSUE THREAD

**Issue by jacob-parnell-rozetta** [google-research/pegasus#71]
Hi, just wondering if there is a trivial way to return the `loss`, `targets_BxT`, `logits_BxTxV` and other tensors defined in `transformer.py` and output them as part of the logging process during training?

**Comment by JingqingZ** [comment]
Hi, could you elaborate on what you mean "trivial"? An example of fetching loss, logits and outputs are in the test https://github.com/google-research/pegasus/blob/master/pegasus/models/transformer_test.py. Tensorboard is also recommended to visualize the loss.

**Comment by jacob-parnell-rozetta** [comment]
I was just curious if additional tensors (eg. `logits` etc.) could be simply output in the logs as part of training along with the `loss` value at each training step during fine-tuning

**Comment by JingqingZ** [comment]
It is possible to return extra values in the model by adding additional elements to the dictionary in https://github.com/google-research/pegasus/blob/939830367bcf411193d2b5eca2f2f90f3f9260ca/pegasus/models/transformer.py#L113

Regarding logging, you may refer to https://github.com/google-research/pegasus/blob/13fcf2b1191e0df950436c82c33d672c1447f5ff/pegasus/eval/text_eval.py#L88 or tensorboard.

**Comment by jacob-parnell-rozetta** [comment]
Great, thank you! I will close the issue.
